src/Component.php

```diff
@@ -128,6 +129,8 @@ protected function loadDefinition(string $pathname) {
   /**
    * Returns the relative file path of the Fractal YAML configuration file for a given component name.
    *
+   * While Fractal do not support file-based variant component configuration files, neither we do.
```

Comment:
Ouch, I first wanted to use "since" but decided to go with "while" 🙈. Thanks for pointing out. I wanted to be clear why this implementation do not support separate config files. So I'd rather exchange the while with a since. Or isn't this ok for you?

Reply:
If both don't do / do something, it's too much "because".  Go with the "neither" statement instead, or simplify furher:

```diff
-Neither Fractal nor this implementation supports separate configuration files per variant.
+Like Fractal, this implementation does not support separate configuration files per variant.
```